src/brainbrowser/surface-viewer/modules/rendering.js

```diff
@@ -634,9 +633,81 @@ BrainBrowser.SurfaceViewer.modules.rendering = function(viewer) {
 
   /**
   * @doc function
+  * @name viewer.rendering:changeCenterRotation
+  * @param {center} a Vector3 that indigate the new center
+  *
+  * @description
+  * Use to change center of rotation.
+  *
+  *
+  * ```js
+  * viewer.changeCenterRotation(center);
+  * ```
+  */
+  viewer.changeCenterRotation = function(center) {
+    var offset     = new THREE.Vector3(0 , 0, 0);
+    // Copy the center into offset, in order to keep center intact
+    // we do not want to manipulate center
+    offset.copy(center);
+    var model      = viewer.model;
+
+    // Adjust the offset value if needed (e.g: if the model was already moved)
+    var offset_old = model.userData.model_center_offset || new THREE.Vector3(0,0,0);
+    offset.x = -offset_old.x - offset.x;
+    offset.y = -offset_old.y - offset.y;
+    offset.z = -offset_old.z - offset.z;
+    offset.negate();
+
+    /*
+      Adjsut all the children.
+    */
+
+    // Translate to original place first
+    model.children.forEach(function(children) {
```

Comment:
I may be wrong, but I think translations should be order insensitive, right? Can't you just do

children.translateX(offset_old.x - offset.x)

rather than performing two loops?